apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -48,13 +47,6 @@ type ModelConfigService interface {
 // FirewallerAPI provides access to the Firewaller API facade.
 type FirewallerAPI struct {
 	*common.LifeGetter
-	*commonmodel.ModelConfigWatcher
-	*common.AgentEntityWatcher
```

Comment:
I believe that this one is a safe remove as it was only ever the application interface on the client using this. We can remove `Watch` from that interface so this won't be used anymore.